src/components/LoginModal/SignupForm/SignupForm.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import LoginLayout from '../LoginLayout/LoginLayout';
+import FormText from '../LoginLayout/FormText/FormText';
+import { api } from '../../../config';
+
+const SignupForm = ({
+  closeLogin,
+  goBack,
+  goToSignin,
+  inputState,
+  handleInput,
+}) => {
+  const history = useHistory();
+
+  // FIXME: 유효성검사 추가 예정
+  // pw: 8자 이상, 대소문자+숫자+특수문자
+
+  const isError =
+    inputState.password &&
+    inputState.passwordConfirmation &&
+    inputState.password !== inputState.passwordConfirmation;
+
+  const submitSignup = async () => {
+    const json = await (
+      await fetch(api.signUp, {
+        method: 'POST',
+        body: JSON.stringify({
+          email: inputState.email,
+          password: inputState.password,
+          'last-name': inputState.lastName,
+          'first-name': inputState.firstName,
```

Comment:
위에 구조분해할당 한 inputState에 이어서, email , password도 줄이면
```suggestion
          email,
          password,
          'last-name': lastName,
          'first-name': firstName,
```

단축속성명으로 줄일 수 있습니다